lib/open_project/translations/models/locales_updater.rb

```diff
@@ -124,4 +128,21 @@ def replace_file(filepath, new_file)
   def js_translation?(translation_file_path)
     !!(translation_file_path.basename.to_s[0..-5] =~ /\Ajs-.+\z/)
   end
+
+  ##
+  # Crowdin has a bug that removes existent translations from new versions
+  # see: https://crowdin.uservoice.com/forums/31787-collaborative-translation-tool/suggestions/8942341-fix-the-recognition-of-duplicated-strings-with-plu
+  def upload_old_translations(plugin_name)
```

Comment:
Unused method argument - `plugin_name`. If it's necessary, use `_` or `_plugin_name` as an argument name to indicate that it won't be used. You can also write as `upload_old_translations(*)` if you want the method to accept any arguments but don't care about them.